hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -290,8 +289,6 @@ public static OMClientRequest createClientRequest(OMRequest omRequest,
       volumeName = keyArgs.getVolumeName();
       bucketName = keyArgs.getBucketName();
       break;
-    case EchoRPC:
-      return new OMEchoRPCRequest(omRequest);
```

Comment:
We can delete the `OmEchoRPCRequest` class

Reply:
Oh sorry i forgot to remove that class! Yes, updated!

Reply:
Thanks ethan for the remind!